app/routes/user.tsx

```diff
@@ -257,7 +257,8 @@ const ProfileContent = () => {
   });
   return (
     <Box sx={{ maxWidth: "600px" }}>
-      <H1>Setup your fundraising profile</H1>
+      <H1 sx={{color: 'color.grey'}}>Setup your fundraising profile</H1>
+      <H1 sx={{color: 'text.secondary'}}>Setup your fundraising profile</H1>
```

Comment:
this is how you could use the theme props defined:
- with the `sx` prop
- through Material's `styled` function, could make `color: text.secondary` for example.

Reply:
Will remove the duplication here before merging